modules/core/src/main/java/org/apache/ignite/internal/visor/verify/VisorValidateIndexesTaskArg.java

```diff
@@ -114,6 +132,14 @@ public int getCheckThrough() {
 
         if (protoVer > V2)
             nodes = U.readSet(in);
+
+        if (protoVer >= V3)
```

Comment:
check it

Reply:
I've added the next version of the protocol.

Reply:
You need to use the next version, in your case V4